CONTRIBUTING.md

```diff
@@ -11,7 +11,23 @@ because pytest does not like spaces in command line arguments.
 
 When a new notebook has a dependency that is not yet supported by the Docker image,
 a new Docker image must be built. Additionally, add the new dependency to the 
-[imports_test notebook](development-notebooks/imports_test.ipynb).
+[imports_test notebook](dev/imports_test.ipynb).
+
+## Area of Interest
+
+The first choice of area of interest (AOI) for any notebook is an AOI that is 
+already used in this repository. The geojson description of these AOIs is given
+in [aois.geojson](dev/imports_test.ipynb) (it is easy to visualize these
+aois directly in GitHub or by copy/pasting into [geojson.io](geojson.io).
+These AOIs are also given in the 
+[repository_aois notebook](dev/repository_aois.ipynb). It is good practice
+to add your notebook to the list of notebooks using each AOI.
+
+If the AOI for a notebook cannot be satisfied by the the AOIs already in use
+in the repository, then email <devrel@planet.com> to request that the demo
+data coverage be expanded to include the new AOI. Once approved, add the AOI to the 
```

Comment:
>  then email <devrel@planet.com> to request that the demo data coverage be expanded to include the new AOI.

let's rephrase this as something like _"then email <devrel...> so that we can consider expanding our demo data coverage to include a new AOI."_ I only say this because I don't want to give the wrong impression about the ease of adding new demo data.

Reply:
Yes! thank you for catching that. changed.